Framework/src/PostProcessingRunner.cxx

```diff
@@ -26,39 +25,25 @@ namespace o2::quality_control::postprocessing
 {
 
 PostProcessingRunner::PostProcessingRunner(std::string name, std::string configPath) //
-  : mConfigFile(ConfigurationFactory::getConfiguration(configPath)),
-    mConfig(name, *mConfigFile)
+  : mName(name), mConfigPath(configPath)
 {
-  // fixme: uncomment after a bugfix in Configuration is merged and released
-  // mConfigFile->setPrefix(""); // protect from having the prefix changed by PostProcessingConfig
 }
 
-PostProcessingRunner::~PostProcessingRunner()
-{
-}
-
-bool PostProcessingRunner::init()
+void PostProcessingRunner::init()
 {
   ILOG(Info) << "Initializing PostProcessingRunner" << ENDM;
 
-  try {
-    // configuration of the database
-    mDatabase = DatabaseFactory::create(mConfigFile->get<std::string>("qc.config.database.implementation"));
-    mDatabase->connect(mConfigFile->getRecursiveMap("qc.config.database"));
-    ILOG(Info) << "Database that is going to be used : " << ENDM;
-    ILOG(Info) << ">> Implementation : " << mConfigFile->get<std::string>("qc.config.database.implementation") << ENDM;
-    ILOG(Info) << ">> Host : " << mConfigFile->get<std::string>("qc.config.database.host") << ENDM;
-    mServices.registerService<DatabaseInterface>(mDatabase.get());
-  } catch (
-    std::string const& e) { // we have to catch here to print the exception because the device will make it disappear todo: is it still the case?
-    ILOG(Error) << "exception : " << e << ENDM;
-    throw;
-  } catch (...) {
-    std::string diagnostic = boost::current_exception_diagnostic_information();
-    ILOG(Error) << "Unexpected exception, diagnostic information follows:\n"
-                << diagnostic << ENDM;
-    throw;
-  }
+  mConfigFile = ConfigurationFactory::getConfiguration(mConfigPath);
+  mConfig = PostProcessingConfig(mName, *mConfigFile);
```

Comment:
It will throw. For `o2-qc-run-postprocessing` exceptions will reach main and the application will return, for `o2-qc-run-postprocessing-occ` the state machine will go into ERROR state.